### PR TITLE
Cockroach CI integration

### PR DESCRIPTION
**Pull request by dpetrick**
Known Prisma Cockroach limitations:
- Json filters`gt`, `gte`, `lt`, `lte`: Not working, missing JSON comparators https://github.com/cockroachdb/cockroach/issues/49144
- `Json[]` type does not work.
- No fulltext search capabilities.
- Enum list operations: `set` works, `push` doesn't (missing concat in CDB https://github.com/cockroachdb/cockroach/issues/71388)
- No `Money` and `Xml` native type.
- Atomic number operations: (Integer) Division doesn't work (https://github.com/cockroachdb/cockroach/issues/41448)
- Enum lists: `isEmpty` operation doesn' work.
- Probably more that will become obvious down the line.

### CODE REVIEW

File: .buildkite/engineer [hunk 1]
```diff
@@ -13,7 +13,7 @@ fi
 if ! type "engineer" > /dev/null; then
     # Setup Prisma engine build & test tool (engineer).
     set -e
-    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/1.29/latest/$OS/engineer.gz" --output engineer.gz
+    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/cockroach/latest/$OS/engineer.gz" --output engineer.gz
```
Comment (dpetrick) [inline .buildkite/engineer:16]: Will change after engineer merge

File: docker-compose.yml [hunk 2]
```diff
@@ -310,9 +310,9 @@ services:
...
-      - 13133:13133  # health check port
-      - 16686:16686  # UI viewer
-      - 4317:55680   # Otel endpoint
+      - 13133:13133 # health check port
+      - 16686:16686 # UI viewer
+      - 4317:55680 # Otel endpoint
 
 networks:
-  databases:
+  databases: null
```
Comment (Weakky) [inline docker-compose.yml:318]: I saw the same change on the engineer PR and should've asked before: what does this do?

**Review by Weakky — APPROVED** [review]
lgtm! ✅

File: docker-compose.yml (hunk 2 above)
Comment (dpetrick) [reply docker-compose.yml:318]: it's just the plugin autocompleting the correct key now. It was always implicitly null, just explicit now.

**Pull request merged**